src/content/contributing/index.mdx

```diff
@@ -50,18 +90,133 @@ All commits must be [signed-off](https://git-scm.com/docs/git-commit#Documentati
 by a user that has signed the
 [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php).
 
+#### Trying out project as end user
+Before contributing to any Open Source project, it's important to have basic understanding of what the project is about.
+It is advised to try out the project as an end user. This project is a collection of Maven and Gradle plugins for [Kubernetes](https://kubernetes.io/)
+and [Red Hat OpenShift](https://www.redhat.com/en/technologies/cloud-computing/openshift), it's advised to learn about these
+technologies if you're interested in seriously contributing to the project.
+
+You can read some blogs about project or watch videos on [Eclipse JKube YouTube channel](https://www.youtube.com/@jkube) .
+
+#### Finding an issue to work on
+
+You're free to start with any type of contribution, whether it is documentation or some issue you faced while
```

Comment:
I would add an `but we advise to start with a first-timers-only issue` somewhere here.

Reply:
I have added this sentence:

![Screenshot_20240517_092134](https://github.com/eclipse-jkube/jkube-website/assets/13834498/b58c161d-c2e2-4b3a-b951-be4b350f323c)